usr/sbin/sbin.mk

```diff
@@ -11,5 +11,7 @@ if ENABLE_CCATOK
 include usr/sbin/pkcscca/pkcscca.mk
 endif
 
+include usr/sbin/pkcstok_migrate/pkcstok_migrate.mk
```

Comment:
Do we want to be able to enable/disable this tool via configure ?

Reply:
Hmm, the tool will only be used when migrating an existing old repository to the 3.12 format. So yes, why always include it. Other tools, like the pkcsep11_migrate tool, are enabled by default when the related token is available. pkcstok_migrate can migrate all token repositories. Well, let's talk about this ...

Reply:
Added config option.